source/position/tests/position_api_tests.cpp

```diff
@@ -0,0 +1,175 @@
+/* Copyright © 2001-2019, Canal TP and/or its affiliates. All rights reserved.
+
+This file is part of Navitia,
+    the software to build cool stuff with public transport.
+
+Hope you'll enjoy and contribute to this project,
+    powered by Canal TP (www.canaltp.fr).
+Help us simplify mobility and open public transport:
+    a non ending quest to the responsive locomotion way of traveling!
+
+LICENCE: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+Stay tuned using
+twitter @navitia
+channel `#navitia` on riot https://riot.im/app/#/room/#navitia:matrix.org
+https://groups.google.com/d/forum/navitia
+www.navitia.io
+*/
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE position_test
+
+#include <boost/test/unit_test.hpp>
+
+#include "ed/build_helper.h"
+#include "position/position_api.h"
+#include "type/pb_converter.h"
+#include "utils/logger.h"
+#include "type/pt_data.h"
+#include "tests/utils_test.h"
+
+using namespace navitia;
+using boost::posix_time::time_period;
+
+struct logger_initialized {
+    logger_initialized() { navitia::init_logger(); }
+};
+BOOST_GLOBAL_FIXTURE(logger_initialized);
+
+namespace {
+
+class positionTestFixture {
+public:
+    ed::builder b;
+    const type::Data& data;
+
+    positionTestFixture() : b("20210913"), data(*b.data) {
+        b.vj_with_network("network:R", "line:A", "1111111", "", true, "")("stop_area:stop1", "10:00"_t, "10:00"_t)(
+            "stop_area:stop2", "11:00"_t, "11:00"_t);
+        b.vj_with_network("network:R", "line:S", "1111111", "", true, "")("stop_area:stop5", "14:00"_t, "14:00"_t)(
+            "stop_area:stop6", "15:00"_t, "15:00"_t);
```

Comment:
Is it possible to add a send vj in the same line with 3 stops so that we have multi vjs in the same vehiclePosition ?

Reply:
ok